Source/GameEngine/Models/User.cs

```diff
@@ -6,28 +6,27 @@ namespace GameEngine.Library.Models
 {
     public class User
     {
+        public int PlayerID { get; set; }
         public string Name { get; set; }
         public List<Pawn> Pawns;
```

Comment:
Who owns the pawn? The player or the board?

Reply:
Spelare, eftersom den får 4 pjäser med vald färg